### DISCUSSION TITLE
Changing the GTK theme on Windows.

### DISCUSSION THREAD

**Discussion by jatutterow** [quodlibet/quodlibet#3985] [Q&A] [answered]
How do you change the appearance of Quod Libet on Windows? Do you have to select a GTK theme somewhere in the config, or is there a way to set a theme for the entire system on Windows. Also, Qoud Libet uses GTK3 on Windows, right?

**Comment by declension** [comment, marked as answer]
Yes GTK 3+ is required everywhere. I don't know much about GTK+ configuration on Windows but https://www.gtk.org/docs/installations/windows/ might be useful

In QL, there's a theme switcher plugin.  Have you tried that?

**Reply by jatutterow** [reply]
I got it with the plugin switcher by finding where the already installed themes are at.
I had to create the themes folder, but you can put new themes in 
`Quod Libet\share\themes`
and then they'll show up in the theme switcher plugin.

Thanks!
